### PR TITLE
Update Changelog and add a comment for Settings id field

### PR DESCRIPTION
**Pull request by Digitalone1**
Just a comment for the purpose of the id string. Related to #292 

Another minor thing. I'll point it out here since it's not so important. I noticed that you set 1190x670 as start size, but in another part of the code you maximize on startup. Don't know if it's intended to work that way but, if you need to maximize on start, maybe don't need to specify a start size...

### CODE REVIEW

**Comment by GyulyVGC** [comment]
It's actually made on purpose.
If not specified, iced will use its default size (which I don't remember) and the screen will be maximised anyway an instant later.
Specifying a custom initial size has the benefit of setting it back when double clicking on the top most bar of the app.
I'm not aware of a configuration to immediately set it to the full screen size.
Note that it's not even suggested to set it to the screen resolution, since iced uses logical pixels which are not always equivalent to physical ones.

**Comment by Digitalone1** [comment]
Ok. I updated the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ All Sniffnet releases with the relative changes are documented in this file.
 
 ## [UNRELEASED]
 
+- Added app `id` in order to correctly show the icon and app name on Linux Wayland ([#292](https://github.com/GyulyVGC/sniffnet/issues/119))
```
Comment (GyulyVGC) [inline CHANGELOG.md:8]: issue number and link should be consistent :P

File: src/main.rs [hunk 2]
```diff
@@ -101,6 +101,10 @@ pub fn main() -> iced::Result {
 
     print_cli_welcome_message();
 
+    // The id string field is essential for Linux Wayland, especially for correctly showing
+    // the app name and icon on Gnome Shell dash (see issue #292).
+    // Besides the id string should be the same specified for `StartupWMClass` keyword in
+    // `sniffnet.desktop` file for Linux packaging.
```
Comment (GyulyVGC) [inline src/main.rs:107]: The comment is fine, I just think that placing it directly above the id field is better

Comment (GyulyVGC) [reply src/main.rs:107]: Still better if it's a bit shorter

File: CHANGELOG.md (hunk 1 above)
Comment (Digitalone1) [reply CHANGELOG.md]: Yes, fixed.

File: src/main.rs (hunk 2 above)
Comment (Digitalone1) [reply src/main.rs]: See if it's good now.

**Review by GyulyVGC — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (GyulyVGC) [reply src/main.rs]: Approved 👍

**Pull request merged**